tests/sentry/snuba/metrics/test_query.py

```diff
@@ -617,15 +617,15 @@ def test_ensure_interval_set_to_granularity_in_performance_queries():
     assert mq.interval == mq.granularity.granularity
 
 
-@freeze_time("2022-11-03 10:10:00")
+@freeze_time("2022-11-03 10:30:00")
 @pytest.mark.parametrize(
     "granularity, interval, expected_granularity",
     [
         pytest.param(
             86400,
-            7200,
             3600,
-            id="day granularity with 2 hour interval",
```

Comment:
Why did you remove this test?

Reply:
I did it because it is not anymore valid as the interval (7200) is bigger than the difference in start and end time. We could solve this by using different start and end or like i did testing with 3600 which is >= than end - start.

Reply:
I would not remove it, I would just create start and end with difference of 4 hrs, and keep the interval at 2 hr

Reply:
Sure!